evolver/app/main.py

```diff
@@ -53,9 +56,28 @@ async def update_evolver(config: EvolverConfigWithoutDefaults):
     app.state.evolver.config_model.save(app_settings.CONFIG_FILE)
 
 
-@app.get("/schema")
-async def get_schema():
-    return app.state.evolver.schema
+class SchemaResponse(pydantic.BaseModel):
```

Comment:
Could be a good time to create an `evolver/app/model.py` module with this in there